package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Demo of React Hot Reloading + TypeScript",
   "main": "./dist/bundle.js",
   "dependencies": {
-    "react": "^15.3.2",
-    "react-dom": "^15.3.2"
+    "tslib": "1.2.0",
```

Comment:
Why is `tslib` needed when `compilerOptions.importHelpers = false`?
https://github.com/Microsoft/tslib

I'm unfamiliar with `tslib` so some explanation would be appreciated. Thanks!